src/acl/ACL.ts

```diff
@@ -1,22 +1,19 @@
-import type { DB, DBTransaction, LevelPath } from '@matrixai/db';
-import type {
-  PermissionId,
-  PermissionIdString,
-  Permission,
-  VaultActions,
-} from './types';
-import type { NodeId } from '../nodes/types';
-import type { GestaltAction } from '../gestalts/types';
-import type { VaultAction, VaultId } from '../vaults/types';
+import type { DB, LevelPath } from '@matrixai/db';
+import type { PermissionId, PermissionIdString, VaultActions } from './types';
 import type { Ref } from '../types';
+import { DBTransaction } from '@matrixai/db';
 import Logger from '@matrixai/logger';
 import { IdInternal } from '@matrixai/id';
 import {
   CreateDestroyStartStop,
   ready,
 } from '@matrixai/async-init/dist/CreateDestroyStartStop';
+import { Permission } from './types';
 import * as aclUtils from './utils';
 import * as aclErrors from './errors';
+import { VaultAction, VaultId } from '../vaults/types';
```

Comment:
Why was this done? Was it caused by `eslint`? I imagine it's something to do with the decorator features which now causes eslint to say that it's needed at runtime. It's not a big deal, but can you confirm that if you move this back to `import type`, that when running `eslint`, it is saying something like:

```
is used by decorator metadata  @typescript-eslint/consistent-type-imports
```

Reply:
Looks like it was changed in a linting commit 44376279d4b3dc387f2157530840ac4eba7189bf. When I run linting now it seems to be changing it back to type imports.

```
/home/faulkes/matrixCode/polykey/js-polykey/src/acl/ACL.ts
   4:1  error  All imports in the declaration are only used as types. Use `import type`  @typescript-eslint/consistent-type-imports
  11:1  error  All imports in the declaration are only used as types. Use `import type`  @typescript-eslint/consistent-type-imports
  14:1  error  All imports in the declaration are only used as types. Use `import type`  @typescript-eslint/consistent-type-imports
  15:1  error  All imports in the declaration are only used as types. Use `import type`  @typescript-eslint/consistent-type-imports
  16:1  error  All imports in the declaration are only used as types. Use `import type`  @typescript-eslint/consistent-type-imports

```

Reply:
Pushed linting changes.